### PR TITLE
Base package

### PR DESCRIPTION
**Pull request by gmmcal**
Features limited select features with pagination

### CODE REVIEW

File: requirements/requirements-base.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+django<2
```
Comment (beerdeaap) [inline requirements/requirements-base.txt]: No Django 2 support ?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM kpndigital/tox:py27_py35
```
Comment (rodgomes) [inline Dockerfile]: is this a public image?

File: requirements/requirements-base.txt (hunk 1 above)
Comment (gmmcal) [reply requirements/requirements-base.txt]: https://github.com/kpn-digital/django-teradata/issues/2 issue created

File: Dockerfile (hunk 2 above)
Comment (igbun) [reply Dockerfile]: Yes, public: https://hub.docker.com/r/kpndigital/tox/

File: MANIFEST.in [hunk 3]
```diff
@@ -0,0 +1,8 @@
+include README.md
```
Comment (igbun) [inline MANIFEST.in]: README.md? You are submitting REAME.rst

Comment (gmmcal) [reply MANIFEST.in]: I'll try to replace for md, but now it is rst. Changing

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+Usage
+-----
+
+Add to your `settings.py` file.
+
+.. code-block:: python
+
+    DATABASES = {
+        'default': {
+            'ENGINE': 'de.db.teradata',
```
Comment (igbun) [inline README.rst]: We need to remove `de` here.

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+                     "            'HOST': 'database_host',\n"
+                     "            'APP_NAME': 'teradata_app_name',\n"
+                     "            'LOG_CONSOLE': False,\n"
+                     "            'DEFAULT_VERSION': '1.0',\n"
+                     "            'CONNECTION_TYPE': 'odbc',\n"
+                     "            'ODBC_LIB_PATH': 'path_to_odbc_library',\n"
+                     '        }\n'
+                     '    }\n',
+ 'name': 'django-teradata',
+ 'packages': ['de', 'de.db', 'de.db.teradata'],
```
Comment (igbun) [inline setup.py]: Shouldn't it be `['django', 'teradata']`?

**Pull request merged**